### PR TITLE
btcmarkets: fix handle errors

### PR DESCRIPTION
**Pull request by egorbulat98**

### CODE REVIEW

File: ts/src/btcmarkets.ts [hunk 1]
```diff
@@ -155,16 +155,18 @@ export default class btcmarkets extends Exchange {
...
-                '6': DDoSProtection,
-                'InsufficientFund': InsufficientFunds,
-                'InvalidPrice': InvalidOrder,
-                'InvalidAmount': InvalidOrder,
-                'MissingArgument': InvalidOrder,
-                'OrderAlreadyCancelled': InvalidOrder,
-                'OrderNotFound': OrderNotFound,
-                'OrderStatusIsFinal': InvalidOrder,
-                'InvalidPaginationParameter': BadRequest,
+                'exact': {
```
Comment (carlosmiei) [inline ts/src/btcmarkets.ts:158]: should we remove the '3' and '6' exceptions?

File: ts/src/btcmarkets.ts [hunk 2]
```diff
@@ -1303,26 +1305,22 @@ export default class btcmarkets extends Exchange {
         return { 'url': url, 'method': method, 'body': body, 'headers': headers };
     }
 
-    handleErrors (code: int, reason: string, url: string, method: string, headers: Dict, body: string, response, requestHeaders, requestBody) {
+    handleErrors (httpCode: int, reason: string, url: string, method: string, headers: Dict, body: string, response, requestHeaders, requestBody) {
```
Comment (carlosmiei) [inline ts/src/btcmarkets.ts:1308]: also i wouldn't rename the parameter here

File: ts/src/btcmarkets.ts (hunk 1 above)
Comment (egorbulat98) [reply ts/src/btcmarkets.ts:158]: yes
this is old errors from api v2, the current api v3 does not have them
you can check all imposible errors there:
https://docs.btcmarkets.net/#tag/ErrorCodes

File: ts/src/btcmarkets.ts (hunk 2 above)
Comment (egorbulat98) [reply ts/src/btcmarkets.ts]: okay. renamed it back

**Pull request merged**